draft-packages/collapsible/KaizenDraft/Collapsible/__snapshots__/Collapsible.spec.tsx.snap

```diff
@@ -28,10 +28,9 @@ exports[`matches snapshot with everything enabled 1`] = `
             class="icon"
             focusable="false"
             role="presentation"
-            viewBox="0 0 0 0"
```

Comment:
We could make these look more like our svg modules at runtime by using the jest transforms from frontend-ops:

https://github.com/cultureamp/frontend-ops/blob/master/packages/frontend-core/jest/transforms/icon.js

Reply:
Neat, i like that idea. These snaps made me do a double-take. I'll add that now thanks!

Reply:
Scratch that - I'll follow up in a separate PR so this one doesn't get too hairy.